<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherPanel</class>
 <widget class="QDialog" name="TeacherPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teacher Panel</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#TeacherPanel {
        background-color: #fef6f0   ;
        border: 1px solid black;
        border-radius: 10%;
}</string>
  </property>
  <widget class="QPushButton" name="btnClassSchedule">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnClassSchedule {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    padding: 6px;
    font-size: 16px;
}

/* Hover effect */
QPushButton#btnClassSchedule:hover {
    background-color: #ee5253;
}

</string>
   </property>
   <property name="text">
    <string>View Class Schedule</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExamResult">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnExamResult {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    padding: 6px;
    font-size: 16px;
}

/* Hover effect */
QPushButton#btnExamResult:hover {
    background-color: #ee5253;
}

</string>
   </property>
   <property name="text">
    <string>Manage Exam Results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAttendanceViewing">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnAttendanceViewing {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    padding: 6px;
    font-size: 16px;
}

/* Hover effect */
QPushButton#btnAttendanceViewing:hover {
    background-color: #ee5253;
}

</string>
   </property>
   <property name="text">
    <string>View Attendance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExamManagement">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnExamManagement {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    padding: 6px;
    font-size: 16px;
}

/* Hover effect */
QPushButton#btnExamManagement:hover {
    background-color: #ee5253;
}

</string>
   </property>
   <property name="text">
    <string>View Assigned Exams</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
